src/components/learner-credit-management/data/hooks/useBudgetDetailHeaderData.js

```diff
@@ -30,6 +30,7 @@ const assignBudgetStatus = (policy) => {
     startDateStr: policy.subsidyActiveDatetime,
     endDateStr: policy.subsidyExpirationDatetime,
     isBudgetRetired: policy.retired,
+    retiredDateStr: '20 May 2024',
```

Comment:
[nit] is this hard coded data necessary?

Reply:
@brobro10000 Good catch, I'm guessing this is hardcoded because the backend work to ensure the `retired_at` timestamp is not complete yet (https://2u-internal.atlassian.net/browse/ENT-8985), so the frontend has no date data to pull from as is.

This raises a question around sequencing, though; @zwidekalanga what is your intended plan around this PR and the related ENT-8985? Does ENT-8985 need to complete before this PR merges in, or will this gracefully handle null `retired_at` in the short term? That is, if this line changes to:

```jsx
retiredDateStr: policy.retiredAt,
```